mava/configs/network/rnn.yaml

```diff
@@ -27,3 +27,15 @@ critic_network:
     layer_sizes: [128]
     use_layer_norm: False
     activation: relu
+
+q_network:
+  pre_torso:
+    module: mava.networks.MLPTorso
+    layer_sizes: [128]
+    use_layer_norm: False
+    activation: relu
+  post_torso:
```

Comment:
There needs to be some enforced consistency between the hidden state dim and the sizes of the pre-and post torso, but for now it is probably okay to just keep the sizes the same!

Reply:
We do not have to do this anymore, both can actually now be any size we want, since we control the hidden state dimension via the config.

Reply:
I got some consistency errors - but this likely has to do with the fact that my network initialisation is not up to date with how we do things with Hydra!